integration-tests/hibernate-search-orm-elasticsearch/src/test/java/io/quarkus/it/hibernate/search/orm/elasticsearch/devservices/HibernateSearchElasticsearchDevServicesDisabledImplicitlyTest.java

```diff
@@ -33,7 +33,7 @@ public Map<String, String> getConfigOverrides() {
                     "quarkus.hibernate-search-orm.schema-management.strategy", "none",
                     // This version does not matter as long as it's supported by Hibernate Search:
                     // it won't be checked in this test anyway.
-                    "quarkus.hibernate-search-orm.elasticsearch.version", "7.6",
+                    "quarkus.hibernate-search-orm.elasticsearch.version", "8.9",
```

Comment:
I think here the idea was to use an "old" version that didn't match the default one, so maybe use 8.8 ?

Reply:
I don't think that was the idea, it's just that this config property must be set to something at least as precise as `x.y` when the version check is disabled (see below).
I'll add a commit to clarify that.

Reply:
OK, works for me